Removed preview information in key pages.

ylorph opened the pull request.

ChrisChinchilla commented on partials/_install-run.md:
```diff
@@ -99,4 +100,6 @@ brew cask install eventstore
 
 In each case you can run Event Store with the `eventstore` command, and stop it with `Ctrl+c`. To use the default database location you need to use `sudo`, or you can change the location with the `--db` parameter.
 
+To build Event Store from source, refer to the [Event Store README](https://github.com/EventStore/EventStore#mac-os-x).
```
@StarfallProjects This link refers specifically to the macOS build instructions, might be best to link to the general heading?

```suggestion
To build Event Store from source, refer to the [Event Store README](https://github.com/EventStore/EventStore#building-event-store).
```

StarfallProjects replied:
That's for the Mac section of the installation guide. There are links to the Windows build from the Windows section, and Linux from the Linux section.

ChrisChinchilla replied:
Huh, didn't see in the changes. Ok

StarfallProjects commented:
The README update PR is here: https://github.com/EventStore/EventStore/pull/1979

ylorph committed:
Removed preview information in key pages.

ylorph committed:
removed preview page from menu.
keeping the content , as we might want to highlight new (preview) features there.

ylorph committed:
added link to roadmap
